docs/setup/custom-config.mdx

```diff
@@ -44,40 +39,99 @@ python -m mindsdb --no_studio
+
+* `absent` (default): Disables permanent storage and is not recommended to use.
+* `local`: Stores files in a local directory defined with `config['paths']['storage']`.
+* `s3`: Stores files in an Amazon S3 bucket. This option requires the `bucket` parameter that specifies the name of the S3 bucket where files will be stored.
+
+```bash
     "storage_dir": "/home/mindsdb/var",
+```
+
+The `storage_dir` parameter specifies the directory where MindsDB stores all its files, including models, uploaded files, tab content, and the internal SQLite database (if running locally).
+
+If this parameter is not set, the path is determined by the `MINDSDB_STORAGE_DIR` environment variable. MindsDB defaults to creating a `mindsdb` folder in the operating system user's home directory.
+
+```bash
     "paths": {
         "root": "/home/mindsdb/var",
-        "content": "/home/mindsdb/var/content",
-        "storage": "/home/mindsdb/var/storage",
-        "static": "/home/mindsdb/var/static",
-        "tmp": "/home/mindsdb/var/tmp",
-        "log": "/home/mindsdb/var/log",
-        "cache": "/home/mindsdb/var/cache",
-        "locks": "/home/mindsdb/var/locks",
+        "content": "/home/mindsdb/var/content", # optional
+        "storage": "/home/mindsdb/var/storage", # optional
+        "static": "/home/mindsdb/var/static", # optional
+        "tmp": "/home/mindsdb/var/tmp", # optional
+        "cache": "/home/mindsdb/var/cache", # optional
+        "locks": "/home/mindsdb/var/locks", # optional
     },
+```
+
+The `paths` parameter allows users to redefine the file paths for various groups of MindsDB files. By default, all paths are located within the `storage_dir`. If only the `root` path is defined, all other folders will be created within that directory. If this parameter is absent, the value is determined by the `MINDSDB_STORAGE_DIR` environment variable.
+
+The `root` parameter defines the base directory for storing all MindsDB files. It serves the same purpose as the `storage_dir` parameter.
+
+The `content` parameter specifies the directory where user-related files are stored, such as uploaded files, created models, and tab content. The internal SQLite database (if running locally) is stored in the `root` directory instead.
+
+If the `['permanent_storage']['location']` is set to `'local'`, then the `storage` parameter is used to store copies of user files.
+
+The `static` parameter is used to store files for the graphical user interface (GUI) when MindsDB is run locally.
+
+The `tmp` parameter designates a directory for temporary files. Note that the operating system’s default temporary directory may also be used for some temporary files.
+
+If the `['cache']['type']` is set to `'local'`, then the `cache` parameter stores cached files for the most recent predictions. For example, if a model is queried with identical input, the result will be stored in the cache and returned directly on subsequent queries, instead of recalculating the prediction.
```

Comment:
`then the `cache` parameter stores cached files` may be `then the `cache` parameter define folder where stores cached files`?

Reply:
Updated it to:
> then the `cache` parameter defines the location for storing cached files